Deploy/stacks/AI4PublicHealth/Stack_Deployment/stack-data-uploader/config/poi_education_and_health.json

```diff
@@ -0,0 +1,39 @@
+{
+  "database": "postgres",
+  "workspace": "the_world_avatar",
+  "datasetDirectory": "PointsOfInterest",
```

Comment:
This directory is not consistent with the directory the readme is placed in: `Deploy/stacks/AI4PublicHealth/Stack_Deployment/stack-data-uploader/data/poi_education_and_health/vector/readme.txt`

Please fix this. The same applies to the other two PoI datasets.

Reply:
I have added a to-do note to the uploader readme.